src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/FamilyLikelihoods.java

```diff
@@ -0,0 +1,455 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import htsjdk.variant.utils.GeneralUtils;
+import htsjdk.variant.variantcontext.*;
+import org.apache.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.QualityUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.samples.Sample;
+import org.broadinstitute.hellbender.utils.samples.SampleDB;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
+
+import java.util.*;
+
+/**
+ * FamilyLikelihoodsUtils code is based on PhaseByTransmission with added posterior probability calculations
+ */
+
+public final class FamilyLikelihoods {
+
+    private static Logger logger = Logger.getLogger(FamilyLikelihoods.class);
```

Comment:
We've been getting loggers through `LogManager.getLogger`.  Probably should do the same for consistency.